notebooks/105-language-quantize-bert/105-language-quantize-bert.ipynb

```diff
@@ -7,8 +7,8 @@
     "id": "rQc-wXjqrEuR"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def create_data_source():
</pre><p>I believe as previously, it should be in prepare dataset section</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/941/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>